src/Hl7.Fhir.Base/Serialization/engine/PocoSerializationEngine_Json.cs

```diff
@@ -0,0 +1,68 @@
+#nullable enable
+
+using Hl7.Fhir.Introspection;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Utility;
+using System;
+using System.Linq;
+using System.Text.Json;
+
+namespace Hl7.Fhir.Serialization;
+
+internal partial class PocoSerializationEngine
+{
+    private readonly FhirJsonPocoDeserializerSettings? _jsonDeserializerSettings;
+    private readonly FhirJsonPocoSerializerSettings? _jsonSerializerSettings;
+    
+    private BaseFhirJsonPocoDeserializer? _jsonDeserializer;
+    private BaseFhirJsonPocoSerializer? _jsonSerializer;
+    
+    private BaseFhirJsonPocoDeserializer getJsonDeserializer() => 
+        _jsonDeserializer ??= new BaseFhirJsonPocoDeserializer(_inspector, _jsonDeserializerSettings!);
+
+    private BaseFhirJsonPocoSerializer getJsonSerializer() =>
+        _jsonSerializer ??= new BaseFhirJsonPocoSerializer(_inspector.FhirRelease, _jsonSerializerSettings!);
+    
+    /// <inheritdoc />
+    public Resource DeserializeFromJson(string data)
+    {
+        return (Resource)deserializeAndFilterErrors(() =>
+        { 
+            _ = getJsonDeserializer().TryDeserializeResource(data, out var instance, out var issues);
+            return (instance, issues);
+        });
+    }
+    
+    /// <inheritdoc />
+    public Resource DeserializeFromJson(ref Utf8JsonReader reader) => deserializeAndFilterErrors(getJsonDeserializer(), ref reader);
+    
+    /// <inheritdoc />
+    public Base DeserializeObjectFromJson(Type targetType, ref Utf8JsonReader reader) => 
+        deserializeObjectAndFilterErrors(targetType, getJsonDeserializer(), ref reader);
+    
+    // overload necessary since ref structs cannot be captured in the lambda
+    private Resource deserializeAndFilterErrors(BaseFhirJsonPocoDeserializer deserializer, ref Utf8JsonReader reader)
+    {
+        _ = deserializer.TryDeserializeResource(ref reader, out var instance, out var issues);
```

Comment:
Same thing here. Using a try, but no matter the outcome do the same thing triggers me. Can you explain?

Reply:
Copied from previous implementation, we could change this of course